src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,58 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+import { Sex } from '../types/PersonSex';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+  const sexFilter = searchParams.get('sex') || null;
+  const centuriesFilter = searchParams.getAll('century') || [];
```

Comment:
if u dont have century in url getAll gonna return empty array so u dont have to add || []